core/controllers/suggestion.py

```diff
@@ -481,6 +481,66 @@ class SuggestionListHandler(base.BaseHandler):
 
     GET_HANDLER_ERROR_RETURN_TYPE = feconf.HANDLER_TYPE_JSON
 
+    URL_PATH_ARGS_SCHEMAS = {}
+
+    HANDLER_ARGS_SCHEMAS = {
+        'GET': {
+            'suggestion_type': {
+                'schema': {
+                    'type': 'basestring',
+                    'choices': feconf.SUGGESTION_TYPE_CHOICES
+                },
+                'default_value': None
+            },
+            'target_type': {
+                'schema': {
+                    'type': 'basestring',
+                    'choices': feconf.SUGGESTION_TARGET_TYPE_CHOICES
+                },
+                'default_value': None
+            },
+            'target_id': {
+                'schema': {
+                    'type': 'basestring'
+                },
+                'default_value': None
+            },
+            'status': {
+                'schema': {
+                    'type': 'basestring'
+                },
+                'default_value': None
+            },
+            'author_id': {
+                'schema': {
+                    'type': 'basestring'
+                },
+                'default_value': None
+            },
+            'final_reviewer_id': {
+                'schema': {
+                    'type': 'basestring'
+                },
+                'default_value': None
+            },
```

Comment:
Add more validations for the structure of the user ID.

Reply:
done + took out final_reviewer_id since it is not used